apps/examples/testcase/le_tc/kernel/tc_libc_misc.c

```diff
@@ -37,12 +37,24 @@
 
 #define BUFF_SIZE 256
 #define USEC_100 100
+#define ARR_NUM 2
+#define LENGTH 1
```

Comment:
`LENGTH` is too common. Could you add which length at the definition name?

Reply:
Please also consider adding a short description of both macros in the comments please.